site/src/docs-components/icons/IconGrid.tsx

```diff
@@ -4,9 +4,7 @@ import IconTile from "./IconTile"
 
 const styles = require("./IconGrid.scss")
 
-type IconGridProps = {}
-
-const IconGrid: React.SFC<IconGridProps> = props => (
+const IconGrid: React.SFC = props => (
```

Comment:
Flagging this as something to verify

Reply:
I initially thought this was a component 🤦‍♀️ Since it's internal than this update is fine.